docs/FAQ.md

```diff
@@ -0,0 +1,121 @@
+# Frequently Asked Questions
+
+## Basics
+
+### How do I choose between the `windows` and `windows-sys` crates?
+
+In general, `windows-sys` is meant to give users access to raw, zero-overhead bindings to the Windows API, while `windows` is meant as a more idiomatic, and when possible safer alternative. Below is a list of reasons you may choose `windows-sys` or `windows`.
+
+| Reason | `windows` | `windows-sys`|
+| --- | --- | --- |
+| Fast compile times possible are one of your top concerns| | ✅ |
+| You need COM or WinRT support. | ✅ | |
+| You would prefer to use APIs that feel idiomatic to Rust. | ✅ | |
+| Reason | You can't afford the very small cost of overhead | | ✅ |
+
+### Where can I find the API I need?
+
+The documentation for the `windows` crate lives [here](https://microsoft.github.io/windows-docs-rs). Since Windows types and functions usually have unique names, you can simply search for the API you need in the search bar, and the docs for that type will be displayed. This will also let you know what namespace your API lives in, so you can use the proper `use` statement.
+
+### Why is the API I need missing?
+
+The `windows` and `windows-sys` crates are generated from metadata such as the [win32metadata project](https://github.com/microsoft/win32metadata). It could be that the API you are searching for is missing from the metadata. In that case, the best thing to do is to [file an issue](https://github.com/microsoft/win32metadata/issues/new).
+
+### Why are some equivalents to C/C++ macros from the Windows SDK missing?
+
+The `windows` and `windows-sys` crates are generated from metadata such as the [win32metadata project](https://github.com/microsoft/win32metadata). These metadata definitions only include type definitions and function signatures and no actual implementation details. As such C/C++ macros are not included. You may find some equivalents of common C/C++ macros in the `windows` crate.
+
+TODO: list equivalent of some common macros
+
+### Is the nightly compiler needed for some features?
+
+Currently, the `interface` and `implementation` proc macros (used to define and implement COM interfaces) require some nightly features. However, those features are in the process of being stablized and so the `interface` and `implementation` macros should soon be available on stable Rust. See [this issue](https://github.com/microsoft/windows-rs/issues/1523) for more information.
```

Comment:
Think it would be helpful to also note potential errors folks may run into for search hits?

```
error[E0658]: trait bounds other than `Sized` on const fn parameters are unstable
error[E0658]: function pointer casts are not allowed in constant functions
```